config.template.toml

```diff
@@ -81,6 +81,8 @@ workspace_base = "./workspace"
 # List of allowed file extensions for uploads
 #file_uploads_allowed_extensions = [".*"]
 
+#approved_hostnames = ["*"]
```

Comment:
In this file, the initial values are typically the fallback values, that is, the user can see here what is currently set, to modify if needed. So we should have 'localhost' here.

Reply:
Done!